CI/check-services.py

```diff
@@ -0,0 +1,168 @@
+import json
```

Comment:
See comment above - wrap main code in a `main` function, also handle file exceptions.